core/jazz_acl/components/casbin.js

```diff
@@ -111,9 +111,11 @@ async function addOrRemovePolicy(serviceId, config, action, policies) {
 
     if (getPolicies && getPolicies.error) {//if there was any error capture that
       result.error = getPolicies.error;
+      return result;
     }
-    else if (getPolicies && !getPolicies.error) {//found previous policies to be removed
-      totalPolicies = getPolicies.filter(policy => policy.length > 0).length;
+
+    totalPolicies = getPolicies ? getPolicies.filter(policy => policy.length > 0).length : 0;
+    if (getPolicies) {//found previous policies to be removed
```

Comment:
If getPolicies value is [[],[],[]], this will not work. Means line num 147 will not execute, which is supposed to execute.